test/test_utils/eks.py

```diff
@@ -381,76 +381,27 @@ def eks_setup():
     run(aws_iam_authenticator_download_command, echo=True)
     run("chmod +x /usr/local/bin/aws-iam-authenticator")
 
-    run(kfctl_download_command, echo=True)
-    run(f"tar -xvf /tmp/kfctl_{KFCTL_VERSION}_{platform.lower()}.tar.gz -C /tmp --strip-components=1")
-    run("mv /tmp/kfctl /usr/local/bin")
-
     run(kubetail_download_command, echo=True)
     run("chmod +x /usr/local/bin/kubetail")
 
     # Run a quick check that the binaries are available in the PATH by listing the 'version'
     run("eksctl version", echo=True)
     run("kubectl version --short --client", echo=True)
     run("aws-iam-authenticator version", echo=True)
-    run("kfctl version", echo=True)
-
 
 def setup_kubeflow(eks_cluster_name,region=os.getenv("AWS_REGION", DEFAULT_REGION)):
-    """Function to setup kubeflow v1.0.2
-        The mxnet operator configuration is not included in the kubeflow v1.0.2 hence installing manually.
-        The mpi operator included in kubeflow v1.0.2 has version v0.1 which has known issues in EKS hence installing the latest
-        version available v0.2
+    """Function to setup kubeflow v1.0.2, MPI and MXNET operators
```

Comment:
avoid version information in docstrings, as it cannot be reliably maintained later.

Reply:
removed